src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Configuration/SftpConfig.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace TrafficCourts.Arc.Dispute.Service.Configuration
```

Comment:
@pbolduc can comment, but I think it's cleaner to convert this namespace declaration to be file-scoped, meaning, just have `namespace TrafficCourts.Arc.Dispute.Service.Configuration;` followed by a semi-colon, drop the brackets and indentation.

Reply:
Not critical. In Visual Studio, if you just add a semi-colon on the end of this statement, it will fix this file.  We can also add this to .editorconfig to make automatic...

```
csharp_style_namespace_declarations = file_scoped
```